Source/WebKit/Shared/mac/WebEventFactory.mm

```diff
@@ -337,6 +344,8 @@ static int typeForEvent(NSEvent *event)
     float deltaX = [event deltaX];
     float deltaY = [event deltaY];
     float deltaZ = [event deltaZ];
+    int unadjustedDeltaX = 0, unadjustedDeltaY = 0;
+    getUnadjustedMovementDelta(event, unadjustedDeltaX, unadjustedDeltaY);
```

Comment:
I think we should just inline the code that sets `unadjustedDeltaX` and `unadjustedDeltaY`. We generally discourage the use of out-parameters, as it makes the code more difficult to follow. Additionally, this method is only used once.

Reply:
Done.